packages/devops/configs/servers.conf

```diff
@@ -234,6 +234,50 @@ server {
         internal;
   }
 
+}
+server {
+
+  listen       443 ssl http2;
+  listen       [::]:443 ssl http2;
+  server_name  ~^.*admin-panel-api\.tupaia\.org$;
+  root         /usr/share/nginx/html;
```

Comment:
Will be a conflict here with @IgorNadj's upcoming changes to move ssl termination to the ELB. Please both watch this and take care to update the other branch after the first merge to dev